ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -42,18 +23,35 @@
       lvm_lvfilesystem: "{{ provision_root_filesystem }}"
 
     # OMERO.web configuration in host_vars in different repository
-    # Upgrade 5.2.8 Web to 5.3.0
     - role: openmicroscopy.omero-web
       omero_web_release: 5.3.2
-      omero_web_upgrade: True
       no_log: true
 
+    # This to be refactored for RHEL/non-spacewalk at some point
```

Comment:
I've found a better way which will work with RHEL/non-spacewalk servers - by checking for the SLS nameservers in the networking config - this was introduced on the `ome-dundeeomero` work and continues. These are good ideas for further refactoring though, so thank you.